.github/workflows/cargo.yml

```
@@ -0,0 +1,29 @@
+name: cargo
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    paths:
+      - "Modules/cpython-sys/**"
```

```suggestion
      - "Modules/cpython-sys/**"
      - "Modules/_base64/**"
```
We should probably also include the base64 acceleration module.

---

Does it make more sense to do `**/*.rs` + `**/Cargo.toml` or other files?

Needing to list every rust-using module here seems like an easy way to get out of date?

---

Actually, thinking about this more, any change to CPython's headers could change the bindings in `cpython-sys`, so I'd say we should probably just not filter and run this on all changes.